libs/core/langchain_core/tracers/evaluation.py

```diff
@@ -38,24 +38,27 @@ class EvaluatorCallbackHandler(BaseTracer):
     """Tracer that runs a run evaluator whenever a run is persisted.
 
     Attributes:
-        example_id : Union[UUID, None]
-            The example ID associated with the runs.
         client : Client
```

Comment:
For some reason, class attributes that don't have a default value don't get picked automatically in attributes list.